digitalocean/database/resource_database_user.go

```diff
@@ -67,6 +78,32 @@ func ResourceDigitalOceanDatabaseUser() *schema.Resource {
 	}
 }
 
+func userACLSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"topic": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"permission": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
On second look, I'm realizing that updates are not actually supported. Combined with the fact that we're not setting state on read as `GetUser` responses not including `settings`, we mislead users into thinking that an update is successfully applied.  So we should add `ForceNew` to these to make it clear that in-place updates are not supported. 

```suggestion
				Type:         schema.TypeString,
				Required:     true,
				ForceNew:     true,
				ValidateFunc: validation.NoZeroValues,
			},
			"permission": {
				Type:     schema.TypeString,
				Required: true,
				ForceNew: true,
```

Reply:
I agree with your suggestion, that makes it more clear that these changes are not allowed in-place. Thanks for the review @andrewsomething !